### ISSUE TITLE
XLNetTokenizer throwing exception

### ISSUE THREAD

**Issue by AbdulRafay** [lonePatient/Bert-Multi-Label-Text-Classification#41]
Hi, 

I am trying to run xlnet and getting and exception at the initialization of `XLNetTokenizer`. I have following files under xlnet>base-cased directory:

1) config.json
2) pytorch_model.bin
3) spiece.model

when i run

`python run_xlnet.py --do_test`

I get the exception:

`not a string`

I have narrowed down the problem to the following line of code:

`self.tokenizer = XLNetTokenizer(vocab_path, do_lower_case)`

@lonePatient  any help in this regard is much appreciated. Thank you

**Comment by lonePatient** [comment]
@AbdulRafay hi, 1. do_train（fine-tuning）， 2. do_test

**Comment by AbdulRafay** [comment]
> @AbdulRafay hi, 1. do_train（fine-tuning）， 2. do_test

@lonePatient I have already trained the model using do_train

**Comment by AbdulRafay** [comment]
> @AbdulRafay hi, 1. do_train（fine-tuning）， 2. do_test

@lonePatient

**Comment by lonePatient** [comment]
@AbdulRafay hi, I konw,,  convert `vocab_path` to `str(vocan_path)` ,and test_data = list(zip(test_data)

**Comment by AbdulRafay** [comment]
@lonePatient that did the trick. Thank you!

**Comment by AbdulRafay** [comment]
@lonePatient is it possible to test just one line of text? currently i saved test.csv to achieve single line testing...

**Comment by lonePatient** [comment]
@AbdulRafay you can run `predict_one.py`

**Comment by AbdulRafay** [comment]
@lonePatient sorry i did not see it there :) Thanks alot for your help. I will close this issue now.
